airflow/models/dagrun.py

```diff
@@ -498,6 +498,37 @@ def fetch_task_instances(
             tis = tis.where(TI.task_id.in_(task_ids))
         return session.scalars(tis).all()
 
+    @internal_api_call
+    def _check_last_N_dagruns_failed(
+        self,
+        dag_id: str,
+        number_of_dag_runs: int,
```

Comment:
Can we just give this argument the same name as the config to avoid variable naming sprawl? E.g. `max_consecutive_failed_dag_runs`, this helps code readability.